test/e2e/propagationpolicy_test.go

```diff
@@ -538,6 +538,7 @@ var _ = ginkgo.Describe("[ImplicitPriority] propagation testing", func() {
 					APIVersion: deployment.APIVersion,
 					Kind:       deployment.Kind,
 					Name:       deployment.Name,
+					Namespace:  deployment.Namespace,
```

Comment:
In fact, there's no need to change it here, because the namespace of the deployment is by default the same as the one of the pp